### PR TITLE
Replace repeated refresh with polling in empty state

### PR DESCRIPTION
**Pull request by CtrlSpice**
App now polls trace store for number of traces every half-second and reloads when there's something there.

### CODE REVIEW

**Comment by jmorrell** [comment]
This is a much better experience! Do you think we could auto-show the most recent trace? or is that a further change?

https://user-images.githubusercontent.com/175496/216736399-d1a7237e-0390-4811-8cb8-da95ae1906ef.mp4

File: desktop-exporter/server.go [hunk 1]
```diff
@@ -25,6 +25,23 @@ type Server struct {
 	traceStore *TraceStore
 }
 
+func traceCountHandler(store *TraceStore) func(http.ResponseWriter, *http.Request){
```
Comment (jmorrell) [inline desktop-exporter/server.go:28]: I'm not sure this needs a new handler. Could we re-use the trace summaries? Then we could redirect to the latest trace directly.

Comment (CtrlSpice) [reply desktop-exporter/server.go:28]: Yes! Good call!

**Comment by CtrlSpice** [comment]
Auto-showing the most recent trace is the thing I'm doing in my next PR 😄

**Review by jmorrell — APPROVED** [review]
DALL·E 2023-02-04 17 23 00 - a corgi floating in vast emptiness, digital art

Note that this needs to be rebased before it is merged.

**Pull request merged**